### ISSUE TITLE
Tables without even/odd shading

### ISSUE THREAD

**Issue by ajneu** [asciidoctor/asciidoctor#1445]
Hello,

I'm trying to write a `readme.adoc` using asciidoctor; and I do _not_ want even/odd shading.

Is there a way of turning it off? I've tried `{set:cellbgcolor:white}` but it doesn't seem to work for the following:

`table_example.adoc`

``` asciidoc
[cols="1,2"]
|===
|*Macro Usage*
a|
[source,cpp]
----
DPRINT("%d %d\n", 1, 2);
----

|*Macro output if `NDEBUG` is not defined* +
  (debugging)
a|[source,cpp]
----
printf("%d %d\n", 1, 2);
----

|*Macro output if `NDEBUG` is defined* +
  (not debugging)
a|
[source,cpp]
----
/* nothing */
----
|===
```

which renders the 2nd ~~column~~ row with an ugly background, that I want to get rid of:
See [rendered-version-link](https://gist.github.com/ajneu/e257c2a7cee604783b3c#file-table_example-adoc)

Thanks.

**Comment by mojavelinux** [comment]
This is a duplicate of #1365. Please refer to that issue to find the workaround that I just posted.

Generally, I don't recommend using the `{set:cellbgcolor:white}`, which may eventually be removed from Asciidoctor in a future version. It compromises the semantics of the document. This type of styling is better handled by the theme (i.e., CSS).

**Comment by mojavelinux** [comment]
The related issue is now resolved. This feature is available starting in 1.5.7.
